internal/service/autoscaling/policy.go

```diff
@@ -134,9 +405,10 @@ func ResourcePolicy() *schema.Resource {
 										},
 									},
 									"predefined_load_metric_specification": {
-										Type:     schema.TypeList,
-										Optional: true,
-										MaxItems: 1,
+										Type:          schema.TypeList,
+										Optional:      true,
+										MaxItems:      1,
+										ConflictsWith: []string{"predictive_scaling_configuration.0.metric_specification.0.customized_load_metric_specification"},
 										Elem: &schema.Resource{
 											Schema: map[string]*schema.Schema{
 												"predefined_metric_type": {
```

Comment:
On line 456 where the strings are hardcoded in the ValidateFunc, we prefer to dive into the SDK and use the list of values defined in the SDK.

ValidateFunc: validation.StringInSlice(autoscaling.PredictiveScalingMode_Values(), false),